using-valtimo/plugin/smartdocuments/configure-smartdocuments.md

```diff
@@ -0,0 +1,46 @@
+# Using the SmartDocuments Plugin
+
+## Configure the plugin
+
+Before using the SmartDocuments Plugin, the plugin has to be configured. A general description on how to configure
+plugins, can be found [here](../configure-plugin.md).
+
+If the SmartDocuments plugin is not visible in the plugin menu, its possible the application is missing a dependency.
+Instructions on how to add the SmartDocuments dependency can be
+found [here](../../../valtimo-implementation/modules/smartdocuments.md)
+
+The 'SmartDocuments URL', 'Username' and 'Password' should all be retrieved from the SmartDocuments application.
+
+An example plugin configuration:
+![example plugin configuration](img/configure-plugin.png)
+
+## Generate documents
+
+SmartDocuments main function is to generate documents based on a template.
+
+### Configuring the process link
+
+A general description on how to create process links, can be found [here](../create-process-link.md).
+
+Both the 'Template group' and 'Template name' can be retrieved from the SmartDocuments application.
+
+The 'Process variable name for storing document' is the name of the process variable where the generated document is
+stored. This process variable can be used to access the document in another BPMN task.
+
+The 'Document format' contains all supported formats for the generated document.
+
+A document is generated with data taken from the 'Template data'. The value on the left-hand side must match the
+placeholder in the SmartDocuments template. The value on the right-hand side can contain any of the following values:
+
+1. A fixed value. This value will be put directly into the template without alterations. For example `John`
```

Comment:
Can you also add this section to `create-process-link.md`?

Reply:
Resolving those placeholder in general is part of another story

Reply:
I think that's one of the stories that's in prep to test, so we need to write the documentation anyway, so that's why I asked. Do you mind writing it?